scripts/update-metadata.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# sync properties in layers config file with live meta data
+# Usage: ./scripts/update-metadata.sh <layerId (optional)>
+
+
+version=0.6.1
+workingDir="./download"
+layersConfigFile="../data/layers-config.json"
+layersConfig=$(cat $layersConfigFile)
+#keys=$(jq -r 'keys[]' <<< "$layersConfig" | head -n 1)
+
+# take optional layer ID from command line argument
+layerID=$1 
+
+# check if layer ID is set
+if [ -z $layerID ]
+then
+    keys=$(jq -r 'keys[]' <<< "$layersConfig")
+else
+    keys=$layerID
+fi
+
+# for all key of the layer config
+for datasetId in ${keys}
+do
+    echo $datasetId 
+
+    # download metadata file
+    path=gs://esa-cfs-tiles/$version/$datasetId/metadata.json
+    metadataFileName=$datasetId-metadata.json
+    gsutil cp $path $workingDir/$metadataFileName
+
+    # iterate over config for dataset
+    props=$(jq -r ".\"$datasetId\" | keys[]"  <<< "$layersConfig")
+    for prop in ${props}
```

Comment:
seems to be possible to merge 2 objects with jq https://stackoverflow.com/a/24904276
but i'm also fine with this solution

Reply:
Well found!

I will give the merge a try tomorrow, would save us the nested loop and fiddling around with tmp files.

Reply:
Done!